ckanext/canada/templates/obd/snippets/search_form.html

```diff
@@ -0,0 +1,56 @@
+{% ckan_extends %}
+{% import 'macros/canada_read.html' as cr %}
+
+{% block search_input_label %}
+
+<div class="col-md-12">
+  <section class="alert alert-info" >
+    {% if h.lang() =='fr' %}
+    <p>L’initiative « Ouvert par défaut » donne accès aux documents de travail utilisés par le gouvernement du Canada. La plupart de ces renseignements ne sont pas publiés. Les documents peuvent être sous la forme d’ébauche et sont en général disponibles dans la langue dans laquelle ils ont été créés. À moins d’indication contraire, les renseignements sont disponibles en vertu de la <a href="http://ouvert.canada.ca/fr/licence-du-gouvernement-ouvert-canada">licence du gouvernement ouvert</a>.</p>
```

Comment:
maybe add a block to public/snippets/search_form so we don't need to copy the whole form to only override the alert that is displayed